README.md

```diff
@@ -43,7 +43,7 @@ webpack-validator makes it easy:
 
 ![validation-example](https://cloud.githubusercontent.com/assets/3755413/14134087/b3279738-f654-11e5-9752-367b01ac123d.png)
 
-### Usage
+### Usage API
```

Comment:
Can you make the changes you want? ATM I am not sure of how to format the infos so everything still looks good and users also see the infos they need on first sight. Maybe a total restructuring would be a good idea. I'll try to make adjustments to the readme in another PR...